dev/Breadcrumb/BreadcrumbBar.xaml

```diff
@@ -336,13 +338,28 @@
                             </Grid>
                             
                         </Button>
+
+                        <ContentPresenter x:Name="PART_LastItemContentPresenter"
```

Comment:
Can you add an interaction test to validate the last element is textblock and does not have the invoke pattern while the other ones do ?